uwblueprint/childrens-aid-society
# Set Up Design System Colors

## Notion ticket link

[Setup Design System Colors](https://www.notion.so/uwblueprintexecs/Set-up-design-system-colors-9f5c11c62c3448c0bac4f74a8cfaabef)



## Implementation description
* Added the colour scheme from the Figma file
* [Figma Colour Scheme File](https://www.figma.com/file/Pi7j5EmcROKxGE3eUFGhDK/%F0%9F%A7%B8-CAS-%2F-Design-System?node-id=39%3A5608)
* Now can be used within the project



## Steps to test
1. Run Localhost:3000 and on the homepage you should see 3 boxes (blue, red and green)
2. Each colour corresponds to the respective colour in the colors.tsx in the theme folder



## What should reviewers focus on?
* See if you can see the colours, matches the colours from the figma file


## Checklist
